managers/file_data_manager/file_data_manager.go

```diff
@@ -0,0 +1,152 @@
+package file_data_manager
```

Comment:
i think that location should be the same in managers folder and a package should be manager too